djangocms_navigation/admin.py

```diff
@@ -102,32 +116,43 @@ def get_preview_link(self, obj):
 
 
 class MenuItemAdmin(TreeAdmin):
+    MenuModel = MenuContent
     form = MenuItemForm
     change_list_template = "admin/djangocms_navigation/menuitem/change_list.html"
     list_display = ["__str__", "get_object_url"]
 
+    def get_admin_name(self, name, reverse=False):
```

Comment:
I would say that this should be called get_admin_reverse_url() as it is returning the url and not an admin name which the naming would suggest.